Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -4228,13 +4235,15 @@ def add_indicators_command(client: Client, args: dict[str, Any]) -> CommandResul
     Returns:
         CommandResults: JSON response of added indicators.
     """
-    result = client.add_indicators(args)
+    result = client.add_indicators(args).get('created_or_updated')
+    indicators = args.get('indicators')
+    feed_uuid = args.get('feed_uuid')
 
     return CommandResults(
         outputs_prefix="SilentPush.AddIndicators",
         outputs_key_field="feed_uuid",
         outputs=result,
-        readable_output=tableToMarkdown("SilentPush Add Indicators", result),
+        readable_output=tableToMarkdown(f"Indicators: '{indicators}' were added successfully to SilentPush feed: '{feed_uuid}'.", result),
```

Comment:
no need for the table 
readable_output=f"Indicators: '{indicators}' were added successfully to SilentPush feed: '{feed_uuid}'."